docs/marketplace-required-csv-annotations.md

```diff
@@ -0,0 +1,85 @@
+# Marketplace Required CSV Annotations
+
+An operator's CSV must contain the following annotations for it to be displayed properly within the Marketplace UI:
+
+```yaml
+metadata:
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
```

Comment:
any reason this is comma separated and not a YAML list, eg:

```
categories:
- cat
  cat2
```

Reply:
I think this is because of the package-server [implementation](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/package-server/apis/packagemanifest/v1alpha1/zz_generated.deepcopy.go#L53). It seems to expect 
```go
map[string]string
```
but I haven't tried adding a list to see what happens - I can try that this afternoon. I agree it would be cleaner (especially on the UI side) to have a list